Dockerfile

```diff
@@ -24,7 +24,7 @@ ENV GOCACHE=/root/.cache/go-build
 # was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO
 # the docker BUILDPLATFORM arg will be linux/arm64 when for Apple x86 it will be linux/amd64. Therefore,
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
-RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager cmd/main.go
+RUN --mount=type=cache,target="/root/.cache/go-build" CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -o manager cmd/main.go
```

Comment:
Do you happen to know why they had the "-a" in there before? Is there some notion that you could somehow have outdated files or something?

Reply:
Is this a best practice for Dockerfiles?

Reply:
> go help build
>   -a force rebuilding of packages that are already up-to-date.

I don't know why they had it in the first place. But looking at the definition of `-a`, it doesn't quite make sense to me to have to do it in this way.